src/MudBlazor/Styles/components/_table.scss

```
@@ -516,7 +516,7 @@
 }
 
 .mud-table-row-expander {
-  margin: -12px -2px -12px -12px;
+  margin: -12px -2px -12px -2px;
```

Try something like this so it flips right/left for you by using start/end

```suggestion
  margin-top: -12px;
  margin-bottom: -12px;
  margin-inline-start: -12px;
  margin-inline-end: -2px;
```